docs/development_guide.md

```diff
@@ -101,6 +164,7 @@ git push -u origin <branch name>
 To commit and push those changes to your branch.
 
 ### Syncing your branch
+
```

Comment:
Are these blank lines needed? If not, consider removing them here and elsewhere.

Reply:
I referred to g3doc style guide, and it suggests newlines before and after the heading. If we do differently, I can remove it.
https://g3doc.corp.google.com/corp/g3doc/docs/reference/style.md#heading_spacing

Reply:
Oh, I see, then maybe consider updating it for every section in this doc.

Reply:
Done.